acceptance.rst

```diff
@@ -0,0 +1,63 @@
+How to Accept a Proposal
+========================
+
+This document describes the technical process for accepting a proposal,
+assuming the proposal has gone through the `proposal process <https://github.com/ghc-proposals/ghc-proposals/#committee-process>`_.
+
+Acceptance checklist
+--------------------
+
+Before accepting, the shepherd should check for the following
```

Comment:
Also: before sending the proposal to the committee to review.

In my view, the first job of the shepherd, when handed a proposal, is to help make sure that the checks below pass. This means both ensuring that the conditions are met before sending the proposal over to the committee, and, when they aren't, to help guide the proposal's authors towards meeting said conditions.

Reply:
Yes, but it's very easy at that stage to focus on the substance of the proposal, instead of these details. And I personally think that's OK. But we should make sure the details are attended to in the end.

Reply:
I guess what I'm getting at is that we should use this opportunity to set the expectations that we have for the shepherd. I'm guessing that we don't have the same image of the role. When I'm shepherding, I try to suspend my judgement as much as possible and focus on ensuring that the proposal is the best proposal it can be, regardless of whether I'm going to recommend acceptance or rejection.